ang/test/civicase/activity/services/activity-feed-measurements.service.spec.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env jasmine */
+(function (_) {
+  describe('ActivityFeedMeasurements', function () {
+    var ActivityFeedMeasurements;
+
+    beforeEach(module('civicase'));
+
+    beforeEach(inject(function (_ActivityFeedMeasurements_) {
+      ActivityFeedMeasurements = _ActivityFeedMeasurements_;
+    }));
+
+    describe('getTopOffset()', function () {
+      var topOffset, expectedTopOffset;
+
+      beforeEach(function () {
+        addAdditionalMarkup();
+
+        topOffset = ActivityFeedMeasurements.getTopOffset();
+        expectedTopOffset = CRM.$('.civicase__activity-feed__body').offset().top + 24;
+      });
+
+      afterEach(function () {
+        removeAdditionalMarkup();
+      });
+
+      it('fetches all contacts of the case', function () {
```

Comment:
is this really the description of the test result?

Reply:
Ooops, apologies. Fixed